x/crosschain/keeper/cctx_gateway_zevm.go

```diff
@@ -0,0 +1,95 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/zeta-chain/zetacore/x/crosschain/types"
+)
+
+type CCTXGatewayZEVM struct {
+	crosschainKeeper Keeper
+}
+
+func NewCCTXGatewayZEVM(crosschainKeeper Keeper) CCTXGatewayZEVM {
+	return CCTXGatewayZEVM{
+		crosschainKeeper: crosschainKeeper,
+	}
+}
+
+/*
+InitiateOutbound handles evm deposit and then ValidateOutbound is called.
+TODO: move remaining of this comment to ValidateOutbound once it's added.
+
+  - If the deposit is successful, the CCTX status is changed to OutboundMined.
+
+  - If the deposit returns an internal error i.e if HandleEVMDeposit() returns an error, but isContractReverted is false, the CCTX status is changed to Aborted.
+
+  - If the deposit is reverted, the function tries to create a revert cctx with status PendingRevert.
+
+  - If the creation of revert tx also fails it changes the status to Aborted.
+
+Note : Aborted CCTXs are not refunded in this function. The refund is done using a separate refunding mechanism.
+We do not return an error from this function , as all changes need to be persisted to the state.
+Instead we use a temporary context to make changes and then commit the context on for the happy path ,i.e cctx is set to OutboundMined.
+*/
+func (c CCTXGatewayZEVM) InitiateOutbound(ctx sdk.Context, cctx *types.CrossChainTx) error {
+	tmpCtx, commit := ctx.CacheContext()
+	isContractReverted, err := c.crosschainKeeper.HandleEVMDeposit(tmpCtx, cctx)
+
+	// further processing will be in validateOutbound(...), for now keeping it here
+	if err != nil && !isContractReverted { // exceptional case; internal error; should abort CCTX
+		cctx.SetAbort(err.Error())
+		return err
+	} else if err != nil && isContractReverted { // contract call reverted; should refund via a revert tx
```

Comment:
```suggestion
	if err != nil && !isContractReverted {
		// exceptional case; internal error; should abort CCTX
		cctx.SetAbort(err.Error())
		return err
	} else if err != nil && isContractReverted {
		// contract call reverted; should refund via a revert tx
```
I know it's been copied but I think we should remove these comments in brackets